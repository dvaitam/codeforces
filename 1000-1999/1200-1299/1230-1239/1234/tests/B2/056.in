12 9
18 17 19 1 20 10 15 5 5 3 19 5
