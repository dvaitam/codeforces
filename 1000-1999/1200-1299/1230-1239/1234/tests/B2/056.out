9
3 5 15 10 20 1 19 17 18
